include/deal.II/base/function_parser.h

```diff
@@ -328,10 +328,17 @@ class FunctionParser : public AutoDerivativeFunction<dim>
 
   /**
    * The muParser objects for each thread (and one for each component). We are
-   * storing unique_ptr so that we don't need to include the definition of
+   * storing shared_ptr so that we don't need to include the definition of
```

Comment:
Since we now switch the pointer type based on the version of `TBB`, shouldn't this statement be more generic? e.g. "We are storing pointers so that..."